rules/REQUEST-903.9006-XENFORO-EXCLUSION-RULES.conf

```diff
@@ -504,6 +508,19 @@ SecRule REQUEST_FILENAME "@endsWith /admin.php" \
         ctl:ruleRemoveTargetById=942340;ARGS:json,\
         ctl:ruleRemoveTargetById=942370;ARGS:json"
 
+# Set forum options
+# POST /xf/admin.php?options/update
+SecRule REQUEST_FILENAME "@endsWith /admin.php" \
+    "id:9006960,\
+    phase:2,\
+    pass,\
+    t:none,\
+    nolog,\
+    chain"
+    SecRule REQUEST_URI "@rx /admin\.php\?options/update" \
+        "t:none,\
+        ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:options[boardInactiveMessage]"
+
```

Comment:
Can we avoid using a chain here?

Reply:
I like matching on the admin.php in REQUEST_FILENAME as just matching on REQUEST_URI could open bypasses (e.g. a call to /foo.php?admin.php%3Foptions/update might be excluded while we don't trust /foo.php). I do notice we could skip it within this section, because we skip over these lines in 9006900, but then the rules are no longer self-contained. But a good case could be made for removing the chain. If we remove the chain here, we should remove it from the other rules as well I suppose.

Reply:
We could write the second rule as `SecRule ARGS_GET "@contains options/update"` to avoid checking the filename again then perhaps?

Reply:
What's wrong with checking the filename two times?

Other than that, I like the contains-idea.

Reply:
Nothing wrong per se, just not really needed.
Also checking the query strings directly means that you cannot bypass it by adding query string parameters that are ignored by the server.

Reply:
To be clear, this is not a stopper. We can update if in a separate PR if people agrees.

Reply:
Removing unnecessary chains will be done in a separate PR.